packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -164,6 +164,18 @@ const LabeledColorIndicators = ( { indicators, label } ) => (
 		<FlexItem
 			className="block-editor-panel-color-gradient-settings__color-name"
 			title={ label }
+			// If it has an inherited value, we want to label with purple background
+			// and purple text.
+			style={
+				hasInheritedValue
+					? {
+							backgroundColor: '#FAF5FE',
+							color: '#7A00DF',
+							borderRadius: 3,
+							padding: '4px 3px',
+					  }
+					: undefined
+			}
```

Comment:
This is interesting, just noting that the "inheritValue" prop is a common prop to all the `*Panel` components and that in global styles for "block panels" it's possible to be set with the value set at the "root level" of global styles for instance.

So basically, the behavior you're introducing here is also going to trigger in the global styles sidebar for blocks. I think that's probably fine but it's something to be aware of.

Reply:
> the behavior you're introducing here is also going to trigger in the global styles sidebar for blocks

yup, thanks for pointing that out! I noticed it as well, but I haven't tried to fix it at all yet.